### PR TITLE
Add Altostra integration

### PR DESCRIPTION
**Pull request by yevk**
### What does this PR do?

Adds the initial Altostra integration.

### Motivation

Add integration listing for Altostra.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

N/A

### CODE REVIEW

File: altostra/manifest.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "aws",
+    "cloud",
+    "configuration & deployment",
+    "log collection"
+  ],
+  "type": "check",
+  "is_public": true,
+  "integration_id": "altostra",
+  "assets": {
+    "dashboards": {},
```
Comment (scseanchow) [inline altostra/manifest.json:29]: Need a direct link to the dashboard path provided.

**Review by scseanchow — COMMENTED** [review]
Would it be possible to create an OOTB dashboard with our log widget so that the user can easily access the logs from Altostra to Datadog?
 
 You can also include a logo and links to your setup page there as well; see best practices: https://datadoghq.dev/integrations-core/guidelines/dashboards/

File: altostra/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Agent Check: altostra
+
+## Overview
+
+Altostra integrates with cloud computing services to provide your development teams with end-to-end workflows. From designing infrastructure and writing code to deploying and managing your applications and services across multiple accounts and environments.
```
Comment (cswatt) [inline altostra/README.md:5]:
```suggestion
Altostra integrates with cloud computing services to provide your development teams with end-to-end workflows.
```

File: altostra/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+With the Altostra Datadog integration, you can automatically instrument your Altostra projects during deployment to send logs and metrics to your Datadog account. Control the integration configuration as policy per deployment-environment and free your engineering teams from manual instrumentation.
```
Comment (cswatt) [inline altostra/README.md:7]:
```suggestion
The Datadog Altostra integration enables you to automatically instrument your Altostra projects during deployment to send logs and metrics to your Datadog account. Control the integration configuration per deployment environment.
```

cleaning up language

File: altostra/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+### Validation
+
+1. Deploy an Altostra project that contains a Lambda function to any environment you've configured for log shipping to Datadog.
+2. Invoke the Lambda function.
+3. You should see the Lambda function logs appear in the _Logs_ view in Datadog.
+
+## Troubleshooting
+
+Need help? Contact Altostra [support@altostra.com](mailto:support@altostra.com).
```
Comment (cswatt) [inline altostra/README.md:36]:
```suggestion
Need help? Contact [Datadog Support][1].

[1]: /help
```

Datadog documentation for community integrations points to Datadog support.

**Review by cswatt — CHANGES_REQUESTED** [review]

File: altostra/manifest.json (hunk 1 above)
Comment (yevk) [reply altostra/manifest.json]: @scseanchow We added a dashboard and linked it in the `manifest.json`.

**Review by FlorianVeaux — APPROVED** [review]

**Pull request merged**